src/components/Controls/index.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import './index.scss'
+import './index.scss';
 
-const Controls = ({isRunning, restart, play, end }) => {
-    return (
-    <div className="controls">
-        <button className={"func restart"} onClick={restart}>
-            <i className="las la-step-backward"></i>
-        </button> 
-        <button className={"func play"} onClick={play}>
-            <i className={`las ${(isRunning ? "la-pause" : "la-play")}`}></i>
-        </button> 
-        <button className="func end" onClick={end}>
-            <i className="las la-step-forward"></i>
-        </button>
-    </div>
-    )
-}
+const Controls = ({ isRunning, restart, play, end }) => {
+	return (
+		<div className="controls">
+			<button className={'func restart'} onClick={restart}>
+				<i className="las la-step-backward"></i>
+			</button>
+			<button className={'func play'} onClick={play}>
+				<i className={`las ${isRunning ? 'la-pause' : 'la-play'}`}></i>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SamooMei/Pomall/pullRequest?prid=7386623)
```suggestion
				<i className={`las ${isRunning ? "la-pause" : 'la-play'}`}></i>
```